src/input-resolvers.test.ts

```diff
@@ -247,7 +256,7 @@ describe('inputFromSearch', () => {
   it('takes keys encoded as URI components', () => {
     const qs = new URLSearchParams()
     qs.append('some%20colors[0]', 'blue')
-    qs.append('some%20colors[1]', 'red%20ish')
```

Comment:
this test case was bogus since `URLSearchParams` will encode values and keys for us.